README.md

```diff
@@ -10,12 +10,12 @@ without implicit ambiguity, unlike in pre-1.0.0 cats or Scalaz 7.
 ## Usage
 
 ```scala
-libraryDependencies += "org.typelevel" %% "cats-mtl" % "1.2.0"
+libraryDependencies += "org.typelevel" %% "cats-mtl" % "1.3.0"
 ```
 
 If your project uses ScalaJS, replace the double-`%` with a triple.  Note that **cats-mtl** has an upstream dependency on **cats-core** version 2.x.
 
-Cross-builds are available for Scala 2.12, 2.13, 3.0.0-M2, 3.0.0-M3, and ScalaJS major version 1.x.
+Cross-builds are available for Scala 2.13, 3.1.3, Scala Native, and ScalaJS major version 1.x.
```

Comment:
I was surprised 2.12 had been dropped, but I just checked, and it's still there...